conda/environments/clx_dev.yml

```diff
@@ -8,12 +8,11 @@ channels:
 - pytorch
 - rapidsai/label/pytorch
 dependencies:
-- cugraph=0.13.*
-- cuxfilter=0.13.*
-- cupy>=6.6.0,<8.0.0a0,!=7.1.0
+- cugraph=0.14.*
```

Comment:
You should add this file to https://github.com/rapidsai/clx/blob/branch-0.14/ci/release/update-version.sh and it will be updated automatically when a new version branch is cut.